backend/src/scripts/utils.js

```diff
@@ -55,4 +62,39 @@ Utils.getCollectionsEndsWith = async (teamspace, str) => {
 	return collections.filter(({ name }) => name.endsWith(str));
 };
 
+const stringSplice = (string, index, length, replacement) => {
+	const before = string.slice(0, index);
```

Comment:
using this config I got 
```

TypeError: string.slice is not a function
    at stringSplice (C:\Dev\3drepo\3drepo.io\backend\src\scripts\utils.js:66:24)
    at Utils.parsePath (C:\Dev\3drepo\3drepo.io\backend\src\scripts\utils.js:83:10)
    at Object.run [as allActiveLicenses] (C:\Dev\3drepo\3drepo.io\backend\src\scripts\utility\teamspaces\allActiveLicenses.js:55:32)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runScripts (C:\Dev\3drepo\3drepo.io\backend\src\scripts\utility\scheduler.js:77:3)
    at async run (C:\Dev\3drepo\3drepo.io\backend\src\scripts\utility\scheduler.js:112:2)
```

```
{
  "daily": [
    {"name": "allActiveLicenses", "params": ["%TEMP%//report.csv"]}, 
    { "name": "cleanUpSharedDir", "params": [5] } ,
    { "name": "viewLicense","params": ["localsignup01"] }
  ],
  "weekly": [
    {"name": "removeIncompleteRevisions", "params": [5]} 
  ],
  "monthly": [
  ]
}

```

Reply:
![image](https://user-images.githubusercontent.com/11945337/219634192-a3cfcbdf-8fd1-4fcd-9c8f-71a7734174fe.png)